### PR TITLE
Proxy fixes

### PR DESCRIPTION
**Pull request by palango**
Use the raiden-contracts `ContractManager` for now and disable version checks.

Also fix some styling and add some type annotations.

### CODE REVIEW

**Comment by pcppcp** [comment]
this might cause some confusion, because two instances of `ChannelManager` will be used in the app

**Comment by palango** [comment]
> this might cause some confusion, because two instances of `ChannelManager` will be used in the app

Yup, that may be. Would you prefer to wait till we remove the old contracts?

**Comment by pcppcp** [comment]
I think we can merge it if we want to use contracts in parallel... we'll eventually switch to wrapped manager when the old contracts are removed.

File: raiden/blockchain/abi.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
 CONTRACT_NETTING_CHANNEL = 'NettingChannelContract'
 CONTRACT_REGISTRY = 'Registry'
 
-CONTRACT_TOKEN_NETWORK_REGISTRY = 'TokenNetworkRegistry'
+CONTRACT_TOKEN_NETWORK_REGISTRY = 'TokenNetworksRegistry'
```
Comment (loredanacirstea) [inline raiden/blockchain/abi.py]: The name should be fixed in the smart contracts, not here, as suggested by Lefteris. Issue: https://github.com/raiden-network/raiden-contracts/issues/107

Comment (LefterisJP) [reply raiden/blockchain/abi.py]: yep, please switch to singular

File: raiden/network/blockchain_service.py [hunk 2]
```diff
@@ -36,8 +36,8 @@ def __init__(
             self,
             privatekey_bin: bytes,
             jsonrpc_client: JSONRPCClient,
-            poll_timeout: int = DEFAULT_POLL_TIMEOUT):
-
+            poll_timeout: int = DEFAULT_POLL_TIMEOUT
```
Comment (LefterisJP) [inline raiden/network/blockchain_service.py]: hehe, you bothered fixing style but forgot the trailing comma in the last argument

File: raiden/network/proxies/secret_registry.py [hunk 3]
```diff
@@ -44,10 +44,12 @@ def __init__(
             CONTRACT_MANAGER.get_contract_abi(CONTRACT_SECRET_REGISTRY),
             to_normalized_address(secret_registry_address),
         )
-        CONTRACT_MANAGER.check_contract_version(
-            proxy.functions.contract_version().call(),
-            CONTRACT_SECRET_REGISTRY
-        )
+
+        # TODO: add this back
```
Comment (LefterisJP) [inline raiden/network/proxies/secret_registry.py]: If you leave a TODO like that please make an issue and assign yourself (or someone else). Same goes for the other instances below

File: raiden/tests/integration/fixtures/blockchain.py [hunk 4]
```diff
@@ -193,6 +202,31 @@ def blockchain_backend(
...
 
+def deploy_contract_web3(
+    contract_name: str,
+    poll_timeout,
+    deploy_client,
+    *args
+):
+    web3 = deploy_client.web3
+
+    contract_interface = CONTRACT_MANAGER.abi['SecretRegistry']
```
Comment (loredanacirstea) [inline raiden/tests/integration/fixtures/blockchain.py]: Please don't use the contract names directly - use the imported names from the `raiden-contracts` package.

File: raiden/network/proxies/secret_registry.py (hunk 3 above)
Comment (palango) [reply raiden/network/proxies/secret_registry.py]: Don't think that's worth an issue right now, added a task here: https://github.com/raiden-network/raiden/issues/1383

Comment (LefterisJP) [reply raiden/network/proxies/secret_registry.py]: @palango Whatever floats your boat. It's just to keep track of them. If it's part of an issue adding a task is fine. Thank you!

File: raiden/settings.py [hunk 5]
```diff
@@ -7,7 +7,7 @@
 RPC_CACHE_TTL = 600
 CACHE_TTL = 60
 ESTIMATED_BLOCK_TIME = 7
-GAS_LIMIT = 3141592  # Morden's gasLimit.
+GAS_LIMIT = 10 * 10**6
```
Comment (LefterisJP) [inline raiden/settings.py]: It's all fine @palango, so I will merge but why did you increase the default gas limit? The 3.14 mill was really old a value, that's true -- ropsten now has 8 mill. But why 10? Any specific reason? I am curious.

Comment (LefterisJP) [reply raiden/settings.py]: @palango I just saw the commit that explains that you needed bigger gas limit for the new deployment so this is why you started this. How much gas was needed exactly?

Comment (loredanacirstea) [reply raiden/settings.py]: 4.8 mil last I know of - for the `TokenNetworkRegistry`.

**Pull request merged**